lib/taskana-core/src/test/java/acceptance/task/UpdateManualPriorityWithSpiAccTest.java

```diff
@@ -0,0 +1,143 @@
+package acceptance.task;
+
+import static acceptance.DefaultTestEntities.defaultTestClassification;
+import static acceptance.DefaultTestEntities.defaultTestObjectReference;
+import static acceptance.DefaultTestEntities.defaultTestWorkbasket;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import acceptance.priorityservice.TestStaticValuePriorityServiceProvider;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
+import testapi.WithServiceProvider;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.ClassificationSummary;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.spi.priority.api.PriorityServiceProvider;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.task.internal.jobs.helper.TaskUpdatePriorityWorker;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+@WithServiceProvider(
+    serviceProviderInterface = PriorityServiceProvider.class,
+    serviceProviders = TestStaticValuePriorityServiceProvider.class)
+class UpdateManualPriorityWithSpiAccTest {
+
+  @TaskanaInject TaskanaEngine taskanaEngine;
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+  TaskUpdatePriorityWorker worker;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    worker = new TaskUpdatePriorityWorker(taskanaEngine);
```

Comment:
Why do we need this here? This is an internal class.
Can't we use it's functionality through one of our official APIs?
Background: The tests are intended as examples for our customers.

If (for some reason) that's not possible: Please integrate to our test API, so that it can be injected. The idea behind the test API is that we don't explicitly instantiate any TASKANA entity^^

Reply:
Since we don't have a unified way to call any background job, we agreed to leave this as is.